scripts/docker-compose-bundled.yml.template

```diff
@@ -349,6 +349,174 @@ services:
+
+          # --- SPRING CONFIG ---
+          - spring.main.allow-circular-references=true
+          - spring.flyway.enabled=false
+          - spring.flyway.check-location=false
+          - spring.flyway.baselineOnMigrate=true
+
+          - spring.datasource.url=jdbc:postgresql://db.openelis.org:5432/clinlims?currentSchema=clinlims
+          - spring.datasource.username=clinlims
+          - spring.datasource.password=clinlims
+          - spring.datasource.driverClassName=org.postgresql.Driver
+          - spring.datasource.max-active=15
+          - spring.datasource.hikari.maximum-pool-size=10
+
+          - spring.jpa.properties.hibernate.format_sql=false
+          - spring.jpa.properties.hibernate.show_sql=false
+          - spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+          - spring.jpa.properties.hibernate.hbm2ddl.auto=update
+          - spring.jpa.properties.hibernate.search.enabled=true
+
+          # --- HAPI FHIR CONFIG ---
+          - hapi.fhir.openapi_enabled=true
+          - hapi.fhir.fhir_version=R4
+          - hapi.fhir.cr_enabled=false
+          - hapi.fhir.server_address=http://fhir.openelis.org:8080/fhir/
+          - hapi.fhir.allow_external_references=true
+          - hapi.fhir.auto_create_placeholder_reference_targets=true
+          - hapi.fhir.default_pretty_print=false
+          - hapi.fhir.default_page_size=100
+          - hapi.fhir.advanced_lucene_indexing=false
+          - hapi.fhir.bulk_export_enabled=false
+          - hapi.fhir.bulk_import_enabled=false
+          - hapi.fhir.fhirpath_interceptor_enabled=false
+          - hapi.fhir.narrative_enabled=false
+          - hapi.fhir.mdm_enabled=false
+          - hapi.fhir.cors.allow_Credentials=true
+          - hapi.fhir.cors.allowed_origin=*
+          - hapi.fhir.search-coord-core-pool-size=20
+          - hapi.fhir.search-coord-max-pool-size=100
+          - hapi.fhir.search-coord-queue-capacity=200
+          - hapi.fhir.max_page_size=200
+          - hapi.fhir.retain_cached_searches_mins=60
+          - hapi.fhir.reuse_cached_search_results_millis=1
+          - hapi.fhir.inline_resource_storage_below_size=4000
+          - hapi.fhir.subscription.resthook_enabled=true
+
+          # --- TESTER CONFIG ---
+          - hapi.fhir.tester.home.name="OE adjacent FHIR Store"
+          - hapi.fhir.tester.home.server_address=http://fhir.openelis.org:8080/fhir/
+          - hapi.fhir.tester.home.refuse_to_fetch_third_party_urls=false
+          - hapi.fhir.tester.home.fhir_version=R4                         
+
+        
+  frontend.openelis.org:
+      image: itechuw/openelis-global-2-frontend:latest
+      container_name: openelisglobal-front-end
+      platform: linux/amd64
+      networks:
+          - ozone
+      environment:
+          - CHOKIDAR_USEPOLLING=true  
+      depends_on:
+            - oe.openelis.org 
+            - fhir.openelis.org   
+      tty: true
+
+  proxy.openelis.org:
```

Comment:
is it possible to merge this with the `proxy` service already existing in this file.